tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -1058,7 +1058,7 @@ public function testIntegerColumnLimit()
               ->save();
         $columns = $table->getColumns();
         $sqlType = $this->adapter->getSqlType($columns[1]->getType(), $columns[1]->getLimit());
-        $this->assertEquals($this->usingMysql8() ? 11 : $limit, $sqlType['limit']);
+        $this->assertEquals($this->usingMysql8() ? 11.0 : $limit, $sqlType['limit']);
```

Comment:
This feels odd.

Reply:
Changed back to `limit` being an integer.